### PR TITLE
14.0 Mig Website Legal Page

### PR DESCRIPTION
**Pull request by ska-ibees**
Migrated following standard OCA procedure:

Previous Migration script removed
Code updated w.r.t change in Odoo core
Readme updated using oca-gen-addon-readme

### CODE REVIEW

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Review by wytzejan — APPROVED** [review]
Looks good!

**Review by HaraldPanten — APPROVED** [review]

**Comment by ska-ibees** [comment]
May I know the next steps, please?

**Comment by ska-ibees** [comment]
@emagdalenaC2i Sorry to directly ping you here. Just wanted to know if it is normal to wait for so long.

OCA-git-bot added the ready to merge label 10 days ago and still it is not merged yet. 

Thanks

**Comment by emagdalenaC2i** [comment]
@ska-ibees Yes, it is "normal", sometimes... 
I´ve seen PRs created and merged in a matter of minutes, and that is normal too... sometimes

Things you can do:
1) Ask the repo maintainers or other users with merge privileges in this repo
2) Review 3 or 5 existing PRs in this repo or other OCA repos and go to 1) asking for some kind of magnificence ;-)

I will do 1) for you now and let's see

@OCA/website-maintainers Please, could you merge this PR?

**Comment by ska-ibees** [comment]
Thanks for the insightful reply. 👍

**Comment by ska-ibees** [comment]
Failed with below error. 

stderr:

    ERROR: Requested docutils<0.15.1.1,>=0.15.1 from https://files.pythonhosted.org/packages/d4/12/6c3fd74a590c7327c98cae008c11d536029fa9cd7924de477e8cb8804186/docutils-0.15.1-post1.tar.gz#sha256=f33ddb723332c6d6b6d99731ee1fc0c35eb4044a2df5cca1c64c8aa78eaf22cb (from oca-maintainers-tools==0.1.dev998+g1b5c7ad) has different version in metadata: '0.15.1'

    

Check the log at /home/travis/.cache/pre-commit/pre-commit.log

The command "pre-commit run --all --show-diff-on-failure --verbose --color always" exited with 3.

**Review by ivantodorovich — APPROVED** [review]
Code review, LGTM :+1: 

@ska-ibees can you try rebasing on 14.0 to see if it solves the issue?
I see dotfiles have been changed since you opened this PR, so it may be related to that

File: website_legal_page/views/website_legal_main_page.xml
Comment (ivantodorovich) [inline website_legal_page/views/website_legal_main_page.xml:16]:
```suggestion
            <span class="legal_page ml-2 mr-2">
                <a href="/legal">Legal</a>
            </span>
```

I would change this. Just a personal opinion

**Comment by ska-ibees** [comment]
May I know the reason why is it still not merged? Just wanted to confirm if anything pending from my side.

**Comment by ivantodorovich** [comment]
> @ska-ibees can you try rebasing on 14.0 to see if it solves the issue?
I see dotfiles have been changed since you opened this PR, so it may be related to that

Try rebasing on `14.0`. 
It seems when the bot runs the tests, the branch fails.. it could be related to that.

**Comment by ioans73** [comment]
LGTM

**Review by ioans73 — APPROVED** [review]

**Comment by v-garcies** [comment]
Everything seems to be ok, what are we waiting for to merge ...?

**Review by hbrunn — APPROVED** [review]
thanks, this was indeed quite long. Technically, ocabot only adds the ready to merge label when somebody with write access reviews and approves (you see the difference as a green checkmark), and most people filter for this label when looking for PRs to merge.

**Comment by ska-ibees** [comment]
Thanks, everyone for making it possible. Even though it took quite a long to merge but feeling excited as it was my first contribution!!! :)

**Pull request merged**
